### DISCUSSION TITLE
Cyprus Alpha TV

### DISCUSSION THREAD

**Discussion by ROBGagn** [iptv-org/community#966] [Q&A]
Hello,

Anybody who can make auto-updated token based links for this channel from https://alphacyprus.com.cy website (i.e: https://l4.cloudskep.com/alphacyp/acy/playlist.m3u8?wmsAuthSign=c2VydmVyX3RpbWU9OS8xOS8yMDIzIDc6NTc6MDAgUE0maGFzaF92YWx1ZT1aZHlYdmpZcy9BK2VGTnNzeVY5cWtRPT0mdmFsaWRtaW51dGVzPTMw) ?

Thanks :-)

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
